tuote.php

```diff
@@ -425,6 +425,7 @@
       $tkrow = mysql_fetch_assoc($tkresult);
 
       $maara = "";
+      $tkrow['tilikausi_loppu'] = $tkrow['tilikausi_loppu']." 99-99-99";
       $ehto  = " and tapahtuma.laadittu >= '$tkrow[tilikausi_alku]' and tapahtuma.laadittu <= '$tkrow[tilikausi_loppu]' ";
```

Comment:
@satuaalto Itse asiassa tähän voisi suiraan koodata:

     and tapahtuma.laadittu >= '$tkrow[tilikausi_alku] 00:00:00' and tapahtuma.laadittu <= '$tkrow[tilikausi_loppu] 23:59:59'